scripts/test-make-requirements.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-pip install pip-tools==3.6.1
+pip install pip-tools>=3.8
```

Comment:
I think that this is still fragile to this kind of change. Long term, maybe we should run the requirements test after the other tests? that way this comparison could run in the same virtualenv where we have pinned pip-tools.

Reply:
> Long term, maybe we should run the requirements test after the other tests?

Sure, I'll do that in a separate PR

Reply:
https://github.com/dimagi/commcare-hq/pull/24522